### ISSUE TITLE
Task 8: Show the value of M in your UI

### ISSUE THREAD

**Issue by petervanhoef** [petervanhoef/Calculator-Brain#5]
Show the value of M (if set) in your UI somewhere.

**Comment by petervanhoef** [comment]
Hint 8: Required Tasks 7 and 8 are Controller/View tasks ONLY. No additional changes in CalculatorBrain are required for these. For example, if M appears anywhere in your CalculatorBrain code, you’ve violated MVC. The M functionality is purely a UI feature that just happens to use the CalculatorBrain Model’s generic variable-processing functionality.

**Comment by petervanhoef** [comment]
I decided to have a memory display next to the sequence display with the intention to have a memory display with a small width compared to the width of the sequence display.
See [Working with Constraints in Interface Builder](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/AutolayoutPG/WorkingwithConstraintsinInterfaceBuidler.html#//apple_ref/doc/uid/TP40010853-CH10-SW1) for tips how to add a constraint. I added the constraint by control dragging from the memory display to the sequence display. As option I selected Equal Widths. In the Attributes Inspector, I changed the Multiplier.
